modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -696,6 +696,47 @@ paths:
                     description:
                         default response
             tags: ["AssetLibrary"]
+    "/asset-libraries/by-external-reference-code/{externalReferenceCode}/pins":
+        delete:
+            description:
+                Unpins the asset library.
+            operationId: deleteAssetLibraryByExternalReferenceCodePin
```

Comment:
I think it would be better called `deleteAssetLibraryPinByExternalReferenceCode`